src/electron/lib/api/apis/UtilApi.ts

```diff
@@ -68,6 +72,20 @@ export class UtilApi implements ElectronMainApi<UtilApi> {
           },
         ],
       },
+      {
```

Comment:
The design of these settings interfaces might not have been so well desinged by me. Perhaps these data models need to be moved to frontend end Svelte components. Such that each settings component can request its own data and that all the logic doesn't the just happen in the Settings.svelte component. Kinda the idea I'm having rn is to use the Settings.svelte file to organize all the various settings components, but they control/store/request their own data.

Reply:
This we will have to discuss more in detail, dont think it will be that trivial to refactor,so we will have to look if its worth the time now before our demos